types/office-js-preview/index.d.ts

```diff
@@ -9391,6 +9391,9 @@ declare namespace Office {
          * 
          * **Errors**:
          * 
+         * - AttachmentTypeNotSupported: The attachment is an embedded image in Rich Text Format,
+         *                               or it isn't an email or calendar item (e.g., a contact or task item).
```

Comment:
According to the style guide, we should favor "for example" or "such as" instead of "e.g.". I think switching to "(such a a contact or task item)" would be best.

Reply:
Thanks for the feedback. Updated.